src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -760,91 +952,202 @@ const Packages = () => {
     } else {
       dispatch(addRecommendedRepository(epelRepo.data[0]));
     }
-    dispatch(addPackage(isSelectingPackage!));
+    if (isSelectingPackage) {
+      dispatch(addPackage(isSelectingPackage!));
+    }
+    if (isSelectingGroup) {
+      dispatch(addGroup(isSelectingGroup!));
+    }
     setIsRepoModalOpen(!isRepoModalOpen);
   };
 
   const composePkgTable = () => {
-    return transformedPackages
-      .slice(computeStart(), computeEnd())
-      .map((pkg, rowIndex) => (
-        <Tr key={`${pkg.name}-${rowIndex}`} data-testid="package-row">
-          <Td
-            select={{
-              isSelected: packages.some((p) => p.name === pkg.name),
-              rowIndex: rowIndex,
-              onSelect: (event, isSelecting) =>
-                handleSelect(pkg, rowIndex, isSelecting),
-            }}
-          />
-          <Td>{pkg.name}</Td>
-          <Td>
-            {pkg.summary ? (
-              pkg.summary
+    let rows: ReactElement[] = [];
+    rows = rows.concat(
+      transformedGroups
+        .slice(computeStart(), computeEnd())
+        .map((grp, rowIndex) => (
+          <Tr key={`${grp.name}-${rowIndex}`} data-testid="package-row">
+            <Td
+              select={{
+                isSelected: groups.some((g) => g.name === grp.name),
+                rowIndex: rowIndex,
+                onSelect: (event, isSelecting) =>
+                  handleGroupSelect(grp, rowIndex, isSelecting),
+              }}
+            />
+            <Td>
+              @{grp.name}
+              <Popover
+                bodyContent={
+                  <div style={{ height: '40em', overflow: 'scroll' }}>
```

Comment:
Let's add a minimum width for the popover to solve this
![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/6e89a87d-97f9-49d2-977f-8de58fe20c6b)

```suggestion
              <Popover
                minWidth="25rem"
                bodyContent={
                  <div style={{ height: '40em', overflow: 'scroll' }}>
```